Gemfile.lock

```diff
@@ -842,6 +842,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-19
+  x86_64-darwin-20
```

Comment:
Fascinating, I've been seeing Darwin 18 perpetually added to my Gemfile since I pulled down the latest post-Rails 6.1 upgrade:

```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index d3ae2d938..a2159acf2 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -841,6 +841,7 @@ GEM
     zonebie (0.6.1)

 PLATFORMS
+  x86_64-darwin-18
   x86_64-darwin-19
   x86_64-linux
```

When I saw this PR, I thought it would be 18 but looks like you've got 20 being added to yours? 🤔

Reply:
Are you running MacOS 12.x Mohave @vaidehijoshi? That's the MacOS version that Darwin 12.0 corresponds to, which could explain why you're seeing it in your Gemfile.lock. I suspect @jgaskins might have something like `arm_64-darwin-20` as he's running natively on an M1 Mac

Reply:
LOL I'm on Mojave 10.14.5, which would definitely explain it 😂

Reply:
BTW I have macOS Catalina and this is what happens when I run `bundle install` after updating my master with this PR:

```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index a13f2dad9..6dc8c164a 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -842,7 +842,7 @@ GEM

 PLATFORMS
   arm64-darwin-20
-  x86_64-darwin-18
+  x86_64-darwin
   x86_64-darwin-19
   x86_64-darwin-20
   x86_64-linux
```

Reply:
> I suspect @jgaskins might have something like `arm_64-darwin-20` as he's running natively on an M1 Mac

@joshpuetz Hmm, I hadn't thought about this. I don't, and now that you mention it I don't understand why not:

```
➜  forem git:(master) ruby -v && (bundle install > /dev/null) && git status
ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [arm64-darwin20]
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
```